src/app/components/algorithms/dialogs/add-algorithm-dialog.component.ts

```diff
@@ -19,10 +19,10 @@ export class AddAlgorithmDialogComponent implements OnInit {
     public dialog: MatDialog
   ) {}
 
-  get name() {
+  get name(): any {
     return this.algorithmForm.get('name');
   }
-  get computationModel() {
+  get computationModel(): any {
```

Comment:
Same here, I think the return type is `AbstractControl | null`

Reply:
Done